governance/polls/Vault Compensation Denomination - April 13, 2020.md

```diff
@@ -0,0 +1,65 @@
+
+The Maker Foundation Interim Governance Facilitator has placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the community which the community can use to signal whether to provide compensation to vault holders in the vault's token (ETH/BAT) or in DAI
+
+This Governance Poll ([FAQ](https://community-development.makerdao.com/makerdao-scd-faqs/scd-faqs/governance)) will be active for 7 days beginning on Monday, April 13 at 4 PM UTC.
+
+## Review
+
+[An initial poll on whether to compensate vault holders has passed.](https://vote.makerdao.com/polling-proposal/qmwfvvguaf8rz8xwgv2cqnzzt9t5h6epzh17qmk2ue99y4)
+
+This poll poses the choice of which token will be used to compensate, namely ETH/BAT or DAI. When vaults are liquidated, assets are sold for DAI to repay the debt. Any remaining assets are returned to the vault. In brief, in ordinary circumstances, vault holders would receive the type of asset that was originally held in the vault.
+
+Arguments for compensation in ETH/BAT:
+- Least surprising to vault owners
+- Avoids the risk that vault owners feel cheated, compromising the objectives of compensation: happy users, no distortion of symmetry, positive impact on reputation, and future mass adoption.
+Arguments for compensation in DAI:
+- Most likely, both BAT and ETH will have appreciated since the liquidation event. Therefore, a nominal 100% payout will almost certainly result in less value than would be obtained by payment in ETH/BAT. However, the payout amount could be adjusted to approximate the amount of appreciation in the planned subsequent poll.
+- The complexity of conducting the compensation is reduced since fewer tokens are involved.
+- Compensation in DAI avoids DAI/ETH and DAI/BAT volatility risk. The amount of compensation can be fixed to a number in DAI without worrying about the DAI value of ETH and BAT.
+
+To further clarify, we present an example. Suppose there was a vault with 10 ETH of collateral and 692 DAI of debt. The vault came under liquidation when the collaterallization ratio fell to 130% (instead of the requisite 150%) at the price of 90 DAI/ETH.
+
+After the 13% liquidation penalty, the vault owes 782 DAI (692 DAI + 13% of 692 DAI). All of the ETH collateral was auctioned off at 0 DAI/ETH while the debt of 692 DAI was repaid by flop auctions.
+
+At the time of liquidation, 10 ETH sold at 90 DAI/ETH would have yielded 900 DAI. When we subtract the 782 DAI owed, we get **a remainder of 118 DAI or 1.3 ETH** (at the rate of 90 DAI/ETH at the time).
+
+There are two possible scenarios:
+
+1. **If this poll decides that the compensation shall be in ETH**
+   - This vault holder could expect a compensation scaled by 1.3 ETH
+   - To provide compensation, Maker would have to purchase this ETH at the current market rate
+  
+2. **If this poll decides that the compensation shall be in DAI**
+   - This vault holder could expect a compensation scaled by 118 DAI
```

Comment:
If we speak of lost collateral compensation with DAI why do we use this value (118)? We should say *1.3 ETH in DAI equivalent* instead.

Reply:
I disagree. "1.3 ETH in DAI equivalent" is a number that is constantly changing. The point of compensating in DAI is to fix the DAI/ETH exchange rate at some level.